scripts/python/services/GithubService.py

```diff
@@ -0,0 +1,345 @@
+import logging
```

Comment:
Note that nothing in this class refers to a specific organisation, so just a straight Copy/Paste job from the other repository 👀 happy if people want to do a quick double check though 😂